### ISSUE TITLE
SIGFPE raised by `__pypy__.intop.int_mod` with invalid values

### ISSUE THREAD

**Issue by devdanzin** [pypy/pypy#5129]
It's possible to exit PyPy while raising a SIGFPE by calling `__pypy__.intop.int_mod` with invalid values:
```python
Python 3.10.14 (5cccf8d934d7, Nov 17 2024, 04:47:11)
[PyPy 7.3.18-alpha0 with GCC 10.2.1 20210130 (Red Hat 10.2.1-11)] on linux
>>>> import __pypy__.intop
>>>> __pypy__.intop.int_mod(1,0)
Floating point exception
```

Found using [fusil ](https://github.com/devdanzin/fusil) by @vstinner.

**Comment by mattip** [comment]
I wonder why we expose those operations in the `__pypy__` module. I see them in the [Z3 blogpost](https://pypy.org/posts/2022/12/jit-bug-finding-smt-fuzzing.html#motivation-a-jit-miscompilation), are they useful more generally? [The documentation](https://doc.pypy.org/en/latest/__pypy__-module.html#integer-operations-with-overflow) talks about ops with c-like overflow instead of becoming bigints.

**Comment by mattip** [comment]
Fixed in 70978a3212d
